tests/System.IO.Pipelines.Tests/ReadableBufferFacts.cs

```diff
@@ -262,7 +262,7 @@ private unsafe void TestIndexOfWorksForAllLocations(ref ReadableBuffer readBuffe
                 var ptr = (byte*)handle.PinnedPointer;
                 for (int i = 0; i < memory.Length; i++)
                 {
-                    addresses[index++] = ptr++;
+                    addresses[index++] = ptr;
```

Comment:
That was a mistake which I have reverted.
With Pin no longer taking an index, the test needs to be modified.